docs/reference/commandline/swarm_join_token.md

```diff
@@ -0,0 +1,52 @@
+<!--[metadata]>
++++
+title = "swarm join-token"
+description = "The swarm join-token command description and usage"
+keywords = ["swarm, join-token"]
+advisory = "rc"
+[menu.main]
+parent = "smn_cli"
++++
+<![end-metadata]-->
+
+# swarm init
+
+```markdown
+Usage:	docker swarm join-token [--rotate] (worker|manager)
+
+Manage join tokens
+
+Options:
+      --help     Print usage
+      --rotate   Rotate join token
+```
+
+Join tokens are the strings passed to the `--token` flag of `swarm join`. This command allows the
```

Comment:
Join tokens are secrets that determine whether or not a node will join the swarm as a manager node or a worker node. You pass the token using the `--token` flag when you run `swarm join`. You can access the current tokens or rotate the tokens using `swarm join-token`.